### PR TITLE
Mention nightlies + betas + add link to the install script + update issue tracker links

### PR DESCRIPTION
**Pull request by wilzbach**
Issue tracker links are updated analogous to https://github.com/travis-ci/travis-build/pull/1282

See also: https://dlang.org/install.html

### CODE REVIEW

**Comment by plaindocs** [comment]
Just checking that dropping @ibuclaw is intentional, otherwise this looks fine

File: user/languages/d.md [hunk 1]
```diff
@@ -14,8 +14,11 @@ sure to read our [Getting Started](/user/getting-started/) and
 
 D is a community-supported language in Travis CI. If you run into any problems, please report them in the
 [Travis CI issue tracker](https://github.com/travis-ci/travis-ci/issues) and cc
-[@ibuclaw](https://github.com/ibuclaw), [@klickverbot](https://github.com/klickverbot) and
-[@MartinNowak](https://github.com/MartinNowak).
+[@MartinNowak](https://github.com/MartinNowak) and [@wilzbach](https://github.com/klickverbot).
```
Comment (ibuclaw) [inline user/languages/d.md]: Wilzbach -> klickverbot?

File: user/languages/d.md [hunk 2]
```diff
@@ -14,8 +14,11 @@ sure to read our [Getting Started](/user/getting-started/) and
...
+Please report compiler-specific issues at [DMD's issue tracker](https://issues.dlang.org), or
+[LDC's issue tracker](https://github.com/ldc-developers/ldc/issues), or
```
Comment (ibuclaw) [inline user/languages/d.md]: Add gdc's issue tracker?

File: user/languages/d.md (hunk 1 above)
Comment (plaindocs) [reply user/languages/d.md]: Seems so.

**Comment by wilzbach** [comment]
> Just checking that dropping @ibuclaw is intentional, otherwise this looks fine

Oh sorry that was intentional. As in https://github.com/travis-ci/travis-build/pull/1282 I replaced you and @klickverbot with the respective issue trackers, because I thought that redirecting the user to the issue trackers is easier for your to handle.

> Probably wasn't intended,

Well I was assuming that you already were okay with this and considering that https://github.com/travis-ci/travis-build/pull/1282 went by without your nor @klickverbot's feedback, I assumed that you guys have more important things to do.
Anyhow, if you want to continue to get the GH spam, please just say so and I revert this.

**Comment by ibuclaw** [comment]
I suppose its fine so long as there's a link to the issue tracker.

**Pull request merged**